# DP: Fix string expansion: \u and \U for code points U+0080 to U+00FF

--- a/bash/lib/sh/strtrans.c
+++ b/bash/lib/sh/strtrans.c
@@ -144,7 +144,7 @@
 		  *r++ = '\\';	/* c remains unchanged */
 		  break;
 		}
-	      else if (v <= UCHAR_MAX)
+	      else if (v <= SCHAR_MAX)
 		{
 		  c = v;
 		  break;
